department-of-veterans-affairs/caseflow
# Assign page in dropdown for any admin'd judge teams

Connects #12423

### Description
Shows links to assign cases page for any judge team the user is an admin of

### Acceptance Criteria
- [ ] Admins of judge teams can see their administered judge teams in their dropdown
- [ ] Dropdown options link to the correct judge team

### Testing Plan
1. Make BVAACTING an admin for BVAAABSHIRE's judge team
   ```ruby
   judge = User.find_by(css_id: "BVAAABSHIRE")
   acting = User.find_by(css_id: "BVAACTING")
   org = JudgeTeam.for_judge(judge)
   OrganizationsUser.make_user_admin(acting, org)
   ```
1. Sign in as BVAACTING and navigate to their queue
1. Ensure only "Assign BVAACTING team cases" appears in the queue
1. Enable SCM feature toggle
   ```ruby
   FeatureToggle.enable!(:judge_admin_scm)
   ```
1. Refresh the user's queue and ensure the dropdown has both "Assign BVAACTING team cases" and  "Assign BVAAABSHIRE's team cases"
1. Ensure clicking "Assign BVAACTING team cases" goes to /queue/189/assign
1. Ensure clicking "Assign BVAAABSHIRE's team cases" goes to /queue/3/assign

### Notes!
Fixes to "/queue/189/assign" showing the same thing as "/queue/3/assign" will be addressed in a followup PR

### User Facing Changes
<img width="415" alt="Screen Shot 2020-01-13 at 2 48 57 PM" src="https://user-images.githubusercontent.com/45575454/72288433-0ab4f300-3617-11ea-97a6-ae988b598a46.png">
